### DISCUSSION TITLE
Hard to create Bill of Materials when transloco packages list node tools in dependencies.

### DISCUSSION THREAD

**Discussion by freed00m** [ngneat/transloco#806] [Q&A]
Hi

I am trying to analyze my dependency tree, and notice that transloco lists many tools as dependencies yet they are not part of the build output.

Tools such as `glob`, `fs-extra`, `ora` and others.

It's odd to me because most of frontend javascript libraries never depends on node tools but rather list them in devDependcies.

In corporate environment there is often need to analyze codebase dependencies to create sboms.

We can always hardcode some exceptions but I am still interested why this is?

**Comment by shaharkazaz** [comment]
@freed00m Well, they are used with the schematics. Maybe the projects need to be restructured or rethought about where the schematics and their dependencies should be placed.
